aten/src/ATen/native/tags.yaml

```diff
@@ -16,3 +16,11 @@
   desc: |
           This tag indicates that the operator doesn't have an explicit entry in
           native_functions.yaml, and instead was generated automatically by the codegen.
+- tag: nondeterministic_seeded
+  desc: |
+          This tag indicates if an operator is nondeterminstically seeded (ie is random)
+          such that the operator intentially produces different results when run twice on the same inputs.
+- tag: nondeterministic_bitwise
```

Comment:
Hmm we shouldn't add this tag unless there's a plan to add this tag to the relevant ops

Reply:
We have `set_nondeterministic_mode` that warns or errors out on bitwise-nondeterministic ops, and it's getting a lot of use, so, if going forward tagging is a common mechanism to classify ops, having `nondeterministic_bitwise` will be useful.